pylint/extensions/docparams.py

```diff
@@ -145,6 +145,11 @@ class DocstringParameterChecker(BaseChecker):
             "useless-type-doc",
             "Please remove the ignored parameter type documentation.",
         ),
+        "W9021": (
+            'Missing any documentation in "%s"',
+            "missing-any-param-doc",
+            "Please add parameter and/or type documentation.",
+        ),
```

Comment:
We could add an old names here, the ideal would be to not force to disable missing-any-param when the old one was already disabled. But they are not really equivalent so maybe you were right to not add it.

Reply:
what does the old names contain exactly?

Reply:
You can search for C0111 (the old name for missing-function-docstring) for an example :)

Reply:
So you say that instead of adding an old names, we can add a warning in the 2.12 what's new file?

Reply:
Yes, something like "A warning might appear if you disabled  ``missing-param-doc``and ``missing-type-doc`` because it became ``missing-any-param-doc`` when there isn't any doc."

Reply:
I added my changes in the what's new for 2.11 and under 2.11.2 in ChangeLog, should I move them to 2.12 then?

Reply:
Yes sorry I did not catch that, this is a "new feature" :)